docs/keybindings/Keybindings_nl.md

```diff
@@ -38,6 +38,7 @@
 <pre>
   <kbd>space</kbd>: uitchecken
   <kbd>o</kbd>: maak een pull-aanvraag
+  <kbd>O</kbd>: maak opties voor pull-aanvragen
```

Comment:
I think this is a better translation:
```
bekijk opties voor pull-aanvraag
```

Reply:
I updated this entry according to suggestions below since they are automatically generated by script. I'm a bit confused whether I should use 

> bekijk opties voor pull-aanvraag

everywhere instead, or this cheat sheet actually can diverge from the code?

Reply:
Good point, i think `bekijk opties voor pull-aanvraag` is the best option